config/webpack/environment.js

```diff
@@ -1,6 +1,27 @@
 const { environment } = require('@rails/webpacker');
 const path = require('path');
 
+const babelLoader = environment.loaders.get('babel');
+
+environment.loaders.insert(
```

Comment:
This one is much better! Can you put this loader in the firefighters project too, please? (remove the current one) It doesn't have to be now though. In the future!!!

Reply:
I saw it but I though it didn't have much support. Yet, it seems to work much better

Reply:
Svg inline loader? We used that in the Oncostats

Reply:
Probably